doc-Installing_on_Microsoft_Azure/topics/installation.adoc

```diff
@@ -14,9 +14,9 @@ After you have completed all the procedures in this guide, you will have a worki
 
 ifdef::miq[]
 . In a browser, navigate to link:https://www.manageiq.org/download[].
-. Select *Microsoft Azure* from the *--Choose your platform--* list.
-. Select *Stable* from the *--Choose a release--* list.
-. Follow the instructions to download the appliance.
+. Find the table with the relevant type of release.
+. Select *Microsoft Azure* from the list in the table.
+. Follow the instructions to complete the download of the appliance.
```

Comment:
Clicking the name will download the image and there is no further instruction, so I think those 2 can be combined to something like:

"Click Microsoft Azure from the list to download the appliance"

Reply:
I made the changes you suggested to Microsoft Azure and all of the other platforms as well.